includes/admin/class-wp-job-manager-addons.php

```diff
@@ -38,47 +39,75 @@ public static function instance() {
 	}
 
 	/**
-	 * Handles output of the reports page in admin.
+	 * Call API to get WPJM add-ons
+	 *
+	 * @since  1.30.0
+	 *
+	 * @param  string $category
+	 * @param  string $term
+	 *
+	 * @return array of add-ons
 	 */
-	public function output() {
-
-		if ( false === ( $addons = get_transient( 'wp_job_manager_addons_html' ) ) ) {
-
-			$raw_addons = wp_remote_get(
-				'https://wpjobmanager.com/add-ons/',
-				array(
-					'timeout'     => 10,
-					'redirection' => 5,
-					'sslverify'   => false
-				)
-			);
-
-			if ( ! is_wp_error( $raw_addons ) ) {
-
-				$raw_addons = wp_remote_retrieve_body( $raw_addons );
-
-				// Get Products
-				$dom = new DOMDocument();
-				libxml_use_internal_errors(true);
-				$dom->loadHTML( $raw_addons );
+	private function get_extensions( $category = null, $term = null ) {
```

Comment:
I'm noticing that we're using the terms "add-ons" and "extensions" in the code, more-or-less interchangeably. Is there a distinction? Should we pick one term and stick with it?

Reply:
I kept going back and forth on this one. I generally prefer extensions over add-ons, but not 100% sure themes fit with that. I guess add-ons is more generic. I'll just go back to pure add-ons.